Assets/Scripts/Character/CharacterAnimator.cs

```diff
@@ -81,6 +86,7 @@ public void LateUpdate()
             animator.SetBool("Turning", !moving && !jumpingOrFalling && Mathf.Abs(cameraController.frameRotation) > this.turningDeadZone);
             animator.SetBool("Jumping", jumping);
             animator.SetBool("Falling", falling);
+            animator.SetBool("Long Falling", kcc.FallingTime >= longFallingThreshold);
```

Comment:
Small code change that allows for transitioning to the long falling animation after a set number of seconds.